CONTRIBUTING.md

```diff
@@ -30,3 +36,133 @@ to review and merge.
 For UI changes, please include screenshots to help reviewers.
 
 Once approved, the author may merge the Pull Request.
+
+
+## "Sprint" rituals
+
+As a volunteer team, we don't have strict sprint rituals. We meet Tuesday nights
+at [OpenOakland](https://openoakland.org) and a lot of work gets over the week
+as individuals are available. Team members are generally available on
+[Slack](https://slack.openoakland.org/) (takes a minute to load) in
+`#census-projects`, so don't hesitate to reach out to others outside of
+OpenOakland meetings. Our best work gets done when folks are available, not at
+the meetings.
+
+
+### Kanban board
+
+Because we don't really time-box our work in Sprints, we lean more towards
```

Comment:
I have no idea what “time-box our work” means.

Reply:
In [Scrum](https://en.wikipedia.org/wiki/Scrum_(software_development)) (the most common agile methodology), Sprints are fixed time time-boxed iterations, e.g. two weeks. Usually you'll do a Sprint Planning meeting at the beginning of the Sprint to set the goals and what work you plan to get done in the Sprint.

Given that we are a volunteer organization, time is difficult to predict. Some week's 3 people might put in 1 hour, one week eight people might put in 8 hours. This sentence is trying to explain that we aren't using fixed time iterations.

Would this be clearer?
> Because team availablity is difficult to predict from week to week, we don't plan our work in Sprints (fixed-time iterations) and lean towards Kanban agile methodology.